### ISSUE TITLE
Key word matching subscribe proposal list

### ISSUE THREAD

**Issue by Mannshoch** [EventFahrplan/EventFahrplan#707]
# Acknowledgments
Please check the following boxes with an `x` if they apply:
* [x ] I have read the latest version of the [contribution guide](
  https://github.com/EventFahrplan/EventFahrplan/blob/master/CONTRIBUTING.md) before creating this pull request.
* [ x] I have checked a few other issues labeled `Feature request` to see how they are written.
* [x] The feature I want to propose would be useful for the majority of users, not only for me personally.
* [x] I am aware that _EventFahrplan_ is mostly developed by one person in their unpaid spare time.
* [ ] I can help myself to get this feature implemented or know someone who wants to do it.

# Preliminaries
Many users have an idea what kind of event they would like visit. 

# Current state
I have to search permanent for their key words.

# Target state
I could set up a list of Keywords I'm interested in and get a messgae and a list of found keywords.
Per key word i could choos if it should be found in title or also in description

**Comment by johnjohndoe** [comment]
@Mannshoch Thank you for your feature request. Did you see that the search terms which you type in are remembered?
Parts of your idea can be found in the search concept v2: https://github.com/EventFahrplan/EventFahrplan/issues/3#issuecomment-2452723089

**Comment by Mannshoch** [comment]
Yes, i use them alot. I have a list of seven terms which I check regularly. Especially becaus I hope for some popup events or Workshops.
If I could setup some triggerwords I would enter even more keywords.
I realized that two keywords I have are related to assemblies, which i try to follow.

https://github.com/EventFahrplan/EventFahrplan/issues/3#issuecomment-2452723089
Is interesting. But I see me more in the first point of **"Rough unsorted ideas"** and **"The schedule screen shows the usual tabular layout but only the rooms and session which match the search result."**

**Comment by johnjohndoe** [comment]
Happy new year, @Mannshoch!

Yes, these two points are very related. Thank you for your feedback once again.

For now there is no UX/UI concept for a feature like this and no timeline. Please hang on here - once someone has time to work on the idea the issue will be updated.
